app/styles/_ellipsis.less

```diff
@@ -133,3 +133,8 @@
     z-index: 1;
   }
 }
+
+// static loading... spacing
+.section-loading {
+  margin: 20px 0;
```

Comment:
I'd vote for the loading message style from the screenshot for all pages where it makes sense.